resources/core/templates/tests/test-external-solution.yaml

```diff
@@ -41,7 +41,7 @@ spec:
             - |
               echo 'TESTING start'
               sleep 10
-              /e2e e2e \
+              /e2e e2e-event-mesh \
```

Comment:
We discussed a bit more and thought that having the new event-mesh-alpha test in it's own testdefinition is better, because the shell script will be easier to write and you can distinguish between both tests in case of a failure.